commands/command.py

```diff
@@ -30,6 +30,27 @@ def parse_time_str(time_str):
 
     return None
 
+def check_set_timer_input(input_string):
+    converter = {'s': 0,
+                 'm': 1,
+                 'h': 2
+                }
+    # convert value equal pow function
+    # example hour to seconds <=> 60 * 60  <=> 60^2 = 3600
+    timer_phase = []
+    # phase = value(passed parameter) * converter value
+    value, power = 0, 0
+    for phase in input_string:
+        key = phase[-1]
+        if key in converter:
+            power = converter[key]
+            value = phase[:-1]
+        else:
+            power = 0
+            value = phase
+        timer = int(value) * pow(60, power)
+        timer_phase.append(timer)
```

Comment:
Suggestion add `Exception` for processing invalid input. Eg: `!timer 120ms 60ms 60ms` , int(value) will cause exception.
```suggestion
    try:
      value, power = 0, 0
      for phase in input_string:
          key = phase[-1]
          if key in converter:
              power = converter[key]
              value = phase[:-1]
          else:
              power = 0
              value = phase
          timer = int(value) * pow(60, power)
          timer_phase.append(timer)
    catch Exception:
      return None  
```